client/my-sites/migrate/step-source-select.jsx

```diff
@@ -49,10 +49,24 @@ class StepSourceSelect extends Component {
 
 					switch ( result.site_engine ) {
 						case 'wordpress':
+							if ( result.site_meta.wpcom_site ) {
+								return this.setState( {
+									error: 'This is site is already hosted on WordPress.com',
+									isLoading: false,
+								} );
+							}
```

Comment:
It would be a bit weird to just send the customer to that site without any explanation why. Also it makes sense to fail with an error as they explicitly want this site to move over to their new site and they can't.

Reply:
Ah OK thanks!